src/lib/hooks/multicall.ts

```diff
@@ -22,6 +23,11 @@ export function useSingleCallResult(...args: SkipFirstTwoParams<typeof multicall
   return multicall.hooks.useSingleCallResult(chainId, latestBlock, ...args)
 }
 
+export function useEthSingleCallResult(...args: SkipFirstTwoParams<typeof multicall.hooks.useSingleCallResult>) {
+  const { latestEthBlock } = useCallContext()
```

Comment:
You don't need to call `useCallContext`, you can just call `useEthBlockNumber` - adding that to `useCallContext` is overkill and subscribes to the block updates unnecessarily.